src/OpenTelemetry.Exporter.Console/ConsoleExporter.cs

```diff
@@ -70,7 +71,17 @@ public override Task<ExportResult> ExportAsync(IEnumerable<Activity> activityBat
                         System.Console.WriteLine("Activity Tags");
                         foreach (var tag in activity.TagObjects)
                         {
-                            System.Console.WriteLine($"\t {tag.Key} : {tag.Value}");
+                            if (!(tag.Value is IEnumerable array))
+                            {
+                                System.Console.WriteLine($"\t {tag.Key} : {tag.Value}");
+                            }
+                            else
+                            {
+                                foreach (var item in array)
```

Comment:
I guess this code still have NullRefException if array is `(Array<int>)null`.

Reply:
Testing locally, if tag.value is null:
```csharp
object obj = null;
var array = obj as IEnumerable;
```

That doesn't throw an exception. The array will have `null` value.

Reply:
What would happen when you iterate `foreach (var item in array)`.

Reply:
💥 🔥

Reply:
in this case:
```csharp
 if (!(tag.Value is IEnumerable array))
                            {
                                System.Console.WriteLine($"\t {tag.Key} : {tag.Value}");
                            }
                            else
                            {
                                foreach (var item in array)
                                {
                                    System.Console.WriteLine($"\t {tag.Key} : {item}");
                                }
                            }
```

if `tag.Value` is null, then, it is not an IEnumerable, so it will enter in the first part.

Reply:
Below, the full example:
```csharp
object obj2 = null;
var y = obj2 as IEnumerable;

if (!(obj2 is IEnumerable array))
{
    Console.WriteLine("not an array");
}
else
{
    foreach (var item in array)
    {
        Console.WriteLine("its an array");
    }
}
```

Reply:
Change `object obj2 = null;` to `IEnumerable obj2 = null;`?

Reply:
strangely, when i changed, it still not an IEnumerable LOL!

Reply:
#1014